### ISSUE TITLE
[Testing] lldb po function support broken in Swift

### ISSUE THREAD

**Issue by willlarche** [material-components/material-components-ios#1311]
### Overview
MDFTextAccessibility as imported by MaterialComponents seems to break on its import in lldb's eyes.

### Reproduction steps
Put a breakpoint anywhere in Swift code and run.
When lldb opens, type 'po' + anything.

You will get this message:

> error: in auto-import:
failed to get module 'MaterialComponentsUnitTests' from AST context:
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "Headers/MDFTextAccessibility-umbrella.h"
        ^
>
>~/Documents/iOS/material-components-ios/catalog/Pods/Target Support Files/MDFTextAccessibility/MDFTextAccessibility-umbrella.h:13:9: note: in file included from ~/Documents/iOS/material-components-ios/catalog/Pods/Target Support Files/MDFTextAccessibility/MDFTextAccessibility-umbrella.h:13:
#import "MDFTextAccessibility.h"
        ^
>
>error: ~/Documents/iOS/material-components-ios/catalog/Pods/MDFTextAccessibility/src/MDFTextAccessibility.h:136:53: error: type arguments cannot be applied to non-parameterized class 'NSArray'
>+ (nullable UIColor *)textColorFromChoices:(nonnull NSArray<UIColor *> *)choices
                                                    ^
>
>error: could not build Objective-C module 'MDFTextAccessibility'
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "Headers/MaterialComponents-umbrella.h"
        ^
>
>~/Documents/iOS/material-components-ios/catalog/Pods/Target Support Files/MaterialComponents/MaterialComponents-umbrella.h:13:9: note: in file included from ~/Documents/iOS/material-components-ios/catalog/Pods/Target Support Files/MaterialComponents/MaterialComponents-umbrella.h:13:
#import "MaterialActivityIndicator.h"
        ^
>
>~/Documents/iOS/material-components-ios/components/ActivityIndicator/src/MaterialActivityIndicator.h:17:9: note: in file included from ~/Documents/iOS/material-components-ios/components/ActivityIndicator/src/MaterialActivityIndicator.h:17:
#import "MDCActivityIndicator.h"

### Operating system and device
Xcode 8.3

**Comment by romoore** [comment]
The inspector window is also showing no data for symbols.

**Comment by ianegordon** [comment]
Closing as ignored.  😞 
Please open a new bug if this is still an issue.  😄
